raiden/tests/utils/eth_node.py

```diff
@@ -111,13 +113,19 @@ def geth_to_cmd(node: Dict, datadir: str, chain_id: ChainID, verbosity: str) ->
             value = node[config]
             cmd.extend([f"--{config}", str(value)])
 
-    # dont use the '--dev' flag
+    # Add parameters that are only required in newer versions
+    geth_version_string, _ = subprocess.Popen(
```

Comment:
Do you think it would be an overkill to abstract the invoking of `geth version` in its own function since we also already use it in `check_geth_version_for_tests()` in conftest.py?

Reply:
I think it can be refactored in a better way. I'll give it a try and report back.

Reply:
My other thought didn't work out.

So I don't have any preference regarding your suggestion. The benefits of code reuse and reduced abstraction/indirection even out in my view. If you prefer the change, say so and I'll do it.

Reply:
It's up to you. For me this is a nitpick on my side. I would do it but I have no strong opinions here. Will approve the PR so that you are free to proceed as you wish.